### ISSUE TITLE
[bug] can't run pipeline

### ISSUE THREAD

**Issue by samlipworth** [bactopia/bactopia#327]
Trying to run a single test sample but getting nowhere - is this a nextflow issue?

**Steps to Reproduce**
```
bactopia --sample test2 --R1 e6489b05-6a26-43a9-a566-4433c8cfa093_1.fastq.gz --R2 e6489b05-6a26-43a9-a566-4433c8cfa093_2.fastq.gz --outdir test
Traceback (most recent call last):
  File "/well/bag/users/lipworth/miniconda3/envs/bactopia/bin/bactopia-download.py", line 515, in <module>
    workflow_modules = parse_workflows(bactopia_path, include_merlin=include_merlin, build_all=args.build_all)
  File "/well/bag/users/lipworth/miniconda3/envs/bactopia/bin/bactopia-download.py", line 146, in parse_workflows
    nf_config, stderr = execute(f'nextflow config -flat {bactopia_path}/main.nf', capture=True)
TypeError: cannot unpack non-iterable NoneType object
/well/bag/users/lipworth/miniconda3/envs/bactopia/bin/nextflow: array assign: line 1: unexpected EOF while looking for matching `"'
/well/bag/users/lipworth/miniconda3/envs/bactopia/bin/nextflow: array assign: line 392: syntax error: unexpected end of file
/well/bag/users/lipworth/miniconda3/envs/bactopia/bin/nextflow: line 155: exec: run: not found

```

**Execution Environment**

- Bactopia Version: 2.0.3
- OS: Ubuntu
- Environment: Cluster/Conda

**Additional Information**
```
# packages in environment at /well/bag/users/lipworth/miniconda3/envs/bactopia:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
_sysroot_linux-64_curr_repodata_hack 3                   h5bd9786_13    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
ariba                     2.14.6           py39h67e14b5_3    bioconda
bactopia                  2.0.3                hdfd78af_0    bioconda
bcftools                  1.14                 h88f3f91_0    bioconda
beautifulsoup4            4.11.1             pyha770c72_0    conda-forge
biopython                 1.79             py39h3811e60_1    conda-forge
blast                     2.12.0               hf3cf87c_4    bioconda
bowtie2                   2.2.5            py39h7cff6ad_8    bioconda
brotli                    1.0.9                h166bdaf_7    conda-forge
brotli-bin                1.0.9                h166bdaf_7    conda-forge
brotlipy                  0.7.0           py39hb9d737c_1004    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2022.5.18.1          ha878542_0    conda-forge
capnproto                 0.9.1                h780b84a_5    conda-forge
cd-hit                    4.8.1                h5b5514e_6    bioconda
certifi                   2022.5.18.1      py39hf3d152e_0    conda-forge
cffi                      1.15.0           py39h4bc2ebd_0    conda-forge
charset-normalizer        2.0.12             pyhd8ed1ab_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
coloredlogs               15.0.1           py39hf3d152e_2    conda-forge
conda                     4.12.0           py39hf3d152e_0    conda-forge
conda-package-handling    1.8.1            py39hb9d737c_1    conda-forge
coreutils                 9.1                  h166bdaf_0    conda-forge
cryptography              37.0.2           py39hd97740a_0    conda-forge
curl                      7.83.1               h7bff187_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dbus                      1.13.18              hb2f20db_0  
dendropy                  4.5.2              pyh3252c3a_0    bioconda
entrez-direct             16.2                 he881be0_0    bioconda
executor                  23.2               pyhd3deb0d_0    conda-forge
expat                     2.4.8                h27087fc_0    conda-forge
fasteners                 0.17.3             pyhd8ed1ab_0    conda-forge
fonttools                 4.33.3           py39hb9d737c_0    conda-forge
freetype                  2.11.0               h70c0345_0  
gettext                   0.21.0               hf68c758_0  
giflib                    5.2.1                h36c2ea0_2    conda-forge
glib                      2.70.2               h780b84a_4    conda-forge
glib-tools                2.70.2               h780b84a_4    conda-forge
gsl                       2.6                  he838d99_2    conda-forge
hmmer                     3.3.2                h87f3376_2    bioconda
htslib                    1.14                 h9753748_2    bioconda
humanfriendly             10.0             py39hf3d152e_2    conda-forge
icu                       70.1                 h27087fc_0    conda-forge
idna                      3.3                pyhd8ed1ab_0    conda-forge
jpeg                      9e                   h166bdaf_1    conda-forge
kernel-headers_linux-64   3.10.0              h4a8ded7_13    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
kiwisolver                1.4.2            py39hf939315_1    conda-forge
krb5                      1.19.3               h3790be6_0    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.38                 h1181459_1  
lerc                      3.0                  h9c3ff4c_0    conda-forge
libarchive                3.5.2                hb890918_2    conda-forge
libblas                   3.9.0           14_linux64_openblas    conda-forge
libbrotlicommon           1.0.9                h166bdaf_7    conda-forge
libbrotlidec              1.0.9                h166bdaf_7    conda-forge
libbrotlienc              1.0.9                h166bdaf_7    conda-forge
libcblas                  3.9.0           14_linux64_openblas    conda-forge
libcurl                   7.83.1               h7bff187_0    conda-forge
libdeflate                1.10                 h7f98852_0    conda-forge
libedit                   3.1.20210910         h7f8727e_0  
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 12.1.0              h8d9b700_16    conda-forge
libgfortran-ng            12.1.0              h69a702a_16    conda-forge
libgfortran5              12.1.0              hdcd56e2_16    conda-forge
libglib                   2.70.2               h174f98d_4    conda-forge
libgomp                   12.1.0              h8d9b700_16    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
libidn2                   2.3.2                h7f98852_0    conda-forge
liblapack                 3.9.0           14_linux64_openblas    conda-forge
libmamba                  0.23.3               hd8a31e3_1    conda-forge
libmambapy                0.23.3           py39hd55135b_1    conda-forge
libnghttp2                1.47.0               h727a467_0    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libopenblas               0.3.20          pthreads_h78a6416_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libsolv                   0.7.22               h6239696_0    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-ng              12.1.0              ha89aaad_16    conda-forge
libtiff                   4.4.0                h0fcbabc_0    conda-forge
libunistring              0.9.10               h7f98852_0    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libwebp                   1.2.2                h3452ae3_0    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.9.14               h22db469_0    conda-forge
libzlib                   1.2.12               h166bdaf_0    conda-forge
llvm-openmp               8.0.1                hc9558a2_0    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
mamba                     0.23.3           py39hfa8f2c8_1    conda-forge
mash                      2.3                  he348c14_1    bioconda
matplotlib-base           3.5.2            py39h700656a_0    conda-forge
mummer                    3.23            pl5321h1b792b2_13    bioconda
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
ncbi-amrfinderplus        3.10.23              h6e70893_1    bioconda
ncbi-genome-download      0.3.1              pyh5e36f6f_0    bioconda
ncurses                   6.3                  h27087fc_1    conda-forge
nextflow                  22.04.0              h4a94de4_0    bioconda
numpy                     1.22.4           py39hc58783e_0    conda-forge
openjdk                   11.0.13              h87a67e3_0  
openjpeg                  2.4.0                hb52868f_1    conda-forge
openmp                    8.0.1                         0    conda-forge
openssl                   1.1.1o               h166bdaf_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
perl                      5.32.1          2_h7f98852_perl5    conda-forge
perl-archive-tar          2.40            pl5321hdfd78af_0    bioconda
perl-carp                 1.50            pl5321hd8ed1ab_0    conda-forge
perl-common-sense         3.75            pl5321hdfd78af_0    bioconda
perl-compress-raw-bzip2   2.103           pl5321h87f3376_0    bioconda
perl-compress-raw-zlib    2.105           pl5321h87f3376_0    bioconda
perl-encode               3.17            pl5321hec16e2b_0    bioconda
perl-exporter             5.74            pl5321hd8ed1ab_0    conda-forge
perl-exporter-tiny        1.002002        pl5321hdfd78af_0    bioconda
perl-extutils-makemaker   7.64            pl5321hd8ed1ab_0    conda-forge
perl-io-compress          2.106           pl5321h87f3376_0    bioconda
perl-io-zlib              1.11            pl5321hdfd78af_0    bioconda
perl-json                 4.06            pl5321hdfd78af_0    bioconda
perl-json-xs              2.34            pl5321h9f5acd7_5    bioconda
perl-list-moreutils       0.430           pl5321hdfd78af_0    bioconda
perl-list-moreutils-xs    0.430           pl5321hec16e2b_1    bioconda
perl-parent               0.238           pl5321hd8ed1ab_0    conda-forge
perl-pathtools            3.75            pl5321hec16e2b_3    bioconda
perl-scalar-list-utils    1.62            pl5321hec16e2b_0    bioconda
perl-types-serialiser     1.01            pl5321hdfd78af_0    bioconda
pillow                    9.1.1            py39hae2aec6_1    conda-forge
pip                       22.1.1             pyhd8ed1ab_0    conda-forge
property-manager          3.0                        py_0    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pycosat                   0.6.3           py39hb9d737c_1010    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyfastaq                  3.17.0                     py_2    bioconda
pymummer                  0.10.3                     py_2    bioconda
pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pysam                     0.19.1           py39h5030a8b_0    bioconda
pysocks                   1.7.1            py39hf3d152e_5    conda-forge
python                    3.9.13          h9a8a25e_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
readline                  8.1.2                h7f8727e_1  
reproc                    14.2.4               h295c915_1  
reproc-cpp                14.2.4               h295c915_1  
requests                  2.27.1             pyhd8ed1ab_0    conda-forge
ruamel_yaml               0.15.100         py39h27cfd23_0  
samtools                  1.15                 h3843a85_0    bioconda
sed                       4.8                  he412f7d_0    conda-forge
setuptools                62.3.2           py39hf3d152e_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
soupsieve                 2.3.1              pyhd8ed1ab_0    conda-forge
spades                    3.15.3               h95f258a_0    bioconda
sqlite                    3.38.5               h4ff8645_0    conda-forge
sysroot_linux-64          2.17                h4a8ded7_13    conda-forge
tbb                       2020.2               h4bd325d_4    conda-forge
tk                        8.6.12               h27826a3_0    conda-forge
tqdm                      4.64.0             pyhd8ed1ab_0    conda-forge
tzdata                    2022a                h191b570_0    conda-forge
unicodedata2              14.0.0           py39hb9d737c_1    conda-forge
unzip                     6.0                  h7f98852_3    conda-forge
urllib3                   1.26.9             pyhd8ed1ab_0    conda-forge
verboselogs               1.7                        py_0    conda-forge
wget                      1.21.3               h0b77cf5_0  
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
yaml-cpp                  0.7.0                h27087fc_1    conda-forge
zlib                      1.2.12               h166bdaf_0    conda-forge
zstd                      1.5.2                h8a70e8d_1    conda-forge
```

**Comment by rpetit3** [comment]
hi @samlipworth,

I released v2.1.0 (https://github.com/bactopia/bactopia/releases/tag/v2.1.0) which might be better to use. Please let me know if it fixes the issue you were having!

Also, are you submitting jobs to a cluster?

Cheers,
Robert

**Comment by rpetit3** [comment]
Hi @samlipworth, 

Did you still want to look into this?

Cheers,
Robert

**Comment by rpetit3** [comment]
Hi @samlipworth,

I'm going to go ahead and close this issue, please feel free to reopen. Also, I've released [v2.1.1](https://github.com/bactopia/bactopia/releases/tag/v2.1.1).

Cheers,
Robert
